code/game/objects/structures/crates_lockers/closets/secure/guncabinet.dm

```diff
@@ -10,20 +10,12 @@
 	icon_opened = "base"
 
 /obj/structure/closet/secure_closet/guncabinet/WillContain()
-	var/list/stuff = list(/obj/random/gun_security  = 1, /obj/random/ammo/security = 2)
```

Comment:
This was an intentional design to add variety to security gameplay. Please revert guncabinet changes in this PR

Reply:
Got approval from West.